Gruntfile.js

```diff
@@ -358,6 +358,7 @@ module.exports = function(grunt) {
         cmd:  function() {
           var sharedLibs = [
             'bulk-docs-utils',
+            'lineage',
```

Comment:
Can we generate this array so folders don't get missed in future? For example, using: https://nodejs.org/api/fs.html#fs_fs_readdir_path_options_callback